subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/adb/Device.kt

```diff
@@ -1,17 +1,20 @@
 package com.avito.instrumentation.reservation.adb
 
 import com.avito.instrumentation.util.waitForCondition
+import com.avito.logger.LoggerFactory
 import com.avito.runner.service.worker.device.Serial
 import com.avito.runner.service.worker.device.adb.Adb
 import com.avito.utils.runCommand
 import com.avito.utils.spawnProcess
 import org.funktionale.tries.Try
 import java.io.File
 
-abstract class Device(
-    protected val logger: (String) -> Unit = {}
-) {
+abstract class Device(protected val loggerFactory: LoggerFactory) {
+
+    private val logger = loggerFactory.create("Device")
```

Comment:
Should we append devices serial to the logs?

Reply:
maybe, if we need go that deep; not right now i think
If so, we need to create wrapper or alternative to `LoggerMetadata` with additional parameters, i don't think it should be in tag